docs/how-to/gpu-enabled-mpi.rst

```diff
@@ -0,0 +1,260 @@
+performance for both intra-node and inter-node GPU-to-GPU communication.
+
+The AMD kernel driver exposes remote direct memory access (RDMA) through *PeerDirect* interfaces.
+This allows network interface cards (NICs) to directly read and write to RDMA-capable GPU device
+memory, resulting in high-speed direct memory access (DMA) transfers between GPU and NIC. These
+interfaces are used to optimize inter-node MPI message communication.
+
+The Open MPI project is an open source implementation of the MPI. It's developed and maintained by
+a consortium of academic, research, and industry partners. To compile Open MPI with ROCm support,
+refer to the following sections:
+
+* :ref:`open-mpi-ucx`
+* :ref:`open-mpi-libfabric`
+
+.. _open-mpi-ucx:
+
+ROCm-aware Open MPI on InfiniBand and RoCE networks using UCX
+================================================================
+
+The `Unified Communication Framework <https://www.openucx.org/documentation>`_ (UCX), is an
+open source, cross-platform framework designed to provide a common set of communication
+interfaces for various network programming models and interfaces. UCX uses ROCm technologies to
+implement various network operation primitives. UCX is the standard communication library for
+InfiniBand and RDMA over Converged Ethernet (RoCE) network interconnect. To optimize data
+transfer operations, many MPI libraries, including Open MPI, can leverage UCX internally.
+
+To set up UCX so that it can be used to compile Open MPI, refer to the following instructions.
+
+1. Set environment variables to install all software components in the same base directory. We use the
+home directory in our example, but you can specify a different location if you want.
+
+    .. code-block:: shell
+
+        export INSTALL_DIR=$HOME/ompi_for_gpu
+        export BUILD_DIR=/tmp/ompi_for_gpu_build
+        mkdir -p $BUILD_DIR
+
+2. Instal UCX. To view UCX and ROCm version compatibility, refer to the
+`communication libraries tables <https://rocm.docs.amd.com/en/latest/reference/gpu_libraries/communication.html>`_.
+
+    .. code-block:: shell
+
+        export UCX_DIR=$INSTALL_DIR/ucx
+        cd $BUILD_DIR
+        git clone https://github.com/openucx/ucx.git -b v1.15.x
+        cd ucx
+        ./autogen.sh
+        mkdir build
+        cd build
+        ../configure -prefix=$UCX_DIR \
+            --with-rocm=/opt/rocm
+        make -j $(nproc)
+        make -j $(nproc) install
+
+3. Install Open MPI.
+
+    .. code-block:: shell
+
+        export OMPI_DIR=$INSTALL_DIR/ompi
+        cd $BUILD_DIR
+        git clone --recursive https://github.com/open-mpi/ompi.git \
+            -b v5.0.x
+        cd ompi
+        ./autogen.pl
+        mkdir build
+        cd build
+        ../configure --prefix=$OMPI_DIR --with-ucx=$UCX_DIR \
+            --with-rocm=/opt/rocm
+        make -j $(nproc)
+        make install
+
+.. _rocm-enabled-osu:
+
+ROCm-enabled OSU
```

Comment:
Per our guideline, expand the abbreviation in the heading?

Reply:
OSU seems to be the common usage (I can't even find it expanded on the internet). @edgargabriel thoughts?

Reply:
Perhaps OSU benchmarks? @edgargabriel